keyboards/cannonkeys/devastatingTKL/rules.mk

```diff
@@ -0,0 +1,19 @@
+#MCU Name
+MCU = STM32F072
```

Comment:
Which 072 chip is this, specifically?

#8179

Reply:
CBT6

Reply:
So this should become STM32F072xB i guess

Reply:
This doesn't need to be done now - I will add them to by PR as they are merged :)